docs/en_US/Compressor/CompressionUtils.md

```diff
@@ -120,4 +120,17 @@ from nni.compression.torch.utils.mask_conflict import MaskConflict
 mc = MaskConflict('./resnet18_mask', net, data)
 mc.fix_mask_conflict()
 mc.export('./resnet18_fixed_mask')
-```
\ No newline at end of file
+```
+
+
+### ModelCounter
```

Comment:
ModelCounter -> Model Flops/Parameters Calculator

Reply:
Which one do you think is better? (`Calculator` or `Counter`). I think we should keep consistent of `counter.py` or `calculater.py`

Reply:
let's use Counter then.